VERSION

```diff
@@ -1 +1 @@
-0.10.0
+0.11.0
```

Comment:
Technically, in Spine server we don't have `0.11.00`, we have `0.11.00-SNAPSHOT`.

Therefore, here you should also have `0.11.00-SNAPSHOT`.